client/src/pages/Search.js

```diff
@@ -63,6 +68,17 @@ function Search() {
           <QuerySearchInput handleSearch={handleSearch} />
         </div>
         <div className="grid md:grid-cols-4 grid-cols-2 md:gap-4 sm:gap-2 gap-1 pb-12">
+          {searchPerformed && profiles.length === 0 && !loading && (
```

Comment:
I dont know if you need those last two ampersands after the !loading check

Reply:
Might've been a typo on my part

Reply:
delete rq and see if it works

Reply:
Nope I need them /:

Reply:
can we please switch from .then to await? otherwise, looks good.